src/catalogue/category/catalogueCardView.component.tsx

```diff
@@ -0,0 +1,133 @@
+import {
+  FormControl,
+  Grid,
+  InputLabel,
+  MenuItem,
+  Pagination,
+  Select,
+} from '@mui/material';
+import CatalogueCard from './catalogueCard.component';
+import React from 'react';
+import { CatalogueCategory } from '../../app.types';
+
+export interface CardViewProps {
+  catalogueCategoryData: CatalogueCategory[];
+  onChangeOpenDeleteCategoryDialog: (
+    catalogueCategory: CatalogueCategory
+  ) => void;
+  onChangeOpenEditCategoryDialog: (
+    catalogueCategory: CatalogueCategory
+  ) => void;
+  onChangeOpenSaveAsDialog: (catalogueCategory: CatalogueCategory) => void;
+  handleToggleSelect: (catalogueCategory: CatalogueCategory) => void;
+  selectedCategories: CatalogueCategory[];
+}
+
+function CardView(props: CardViewProps) {
```

Comment:
```suggestion
function CatalogueCardView(props: CatalogueCardViewProps) {
```
Could this be renamed to match the file please